tests/tracking/test_client.py

```diff
@@ -75,3 +81,13 @@ def test_client_create_run_overrides(mock_store):
         entry_point_name=tags[MLFLOW_PROJECT_ENTRY_POINT],
         source_version=tags[MLFLOW_GIT_COMMIT]
     )
+
+
+def test_client_search_runs(mock_store, mock_search_filter):
```

Comment:
Nice, it'd be cool to also add a similar end-to-end test without mocking out the store in `tests/tracking/test_tracking.py` that gets & inspects the search result (e.g. log a metric across two different runs, search, and verify that only one of the runs shows up)

Reply:
Done. Added a couple of tests for single and multiple experiments.